modules/platforms/cpp/ignite/schema/binary_tuple_builder.h

```diff
@@ -431,7 +552,11 @@ class binary_tuple_builder {
      * @return Required size.
      */
     static SizeT gauge_number(const big_decimal &value) noexcept {
-        return SizeT(value.is_zero() ? 0 : value.get_unscaled_value().byte_size());
+        if (value.is_zero()) {
+            return 0;
+        }
+
+        return value.get_unscaled_value().byte_size();
```

Comment:
Hmm. What's the point of this change?

Reply:
Sorry, it's a leftover from the incorrect calculation